scripts/variantstore/TERRA_QUICKSTART.md

```diff
@@ -36,8 +36,8 @@ These are the required parameters which must be supplied to the workflow:
 | project_id            | The name of the google project containing the dataset |
 | dataset_name          | The name of the dataset you created above       |
 | external_sample_names | datamodel  (e.g `this.samples.sample_id`)     |
-| workspace_namespace   | name of the current workspace namespace |
-| workspace_name        | name of the current workspace |
+| workspace_namespace   | name of the current workspace namespace | ## is this still needed?
+| workspace_name        | name of the current workspace | ## is this still needed?
```

Comment:
I don't see either of these inputs in `GvsAssignIds`, so feel free to remove them.